src/components/dialogs/StartPrintDialog.vue

```diff
@@ -1,23 +1,28 @@
 <template>
-    <v-dialog v-model="bool" :max-width="dialogWidth">
+    <v-dialog v-model="bool" :max-width="dialogWidth" @click:outside="closeDialog" @keydown.esc="closeDialog">
         <v-card>
-            <v-img v-if="file.big_thumbnail" contain :src="file.big_thumbnail"></v-img>
-            <v-card-title class="headline">{{ $t('Dialogs.StartPrint.Headline') }}</v-card-title>
+            <v-img v-if="file.big_thumbnail" contain :src="file.big_thumbnail" />
+            <v-card-title class="text-h5">
+                {{ $t('Dialogs.StartPrint.Headline') }}
+            </v-card-title>
             <v-card-text class="pb-0">
-                {{ $t('Dialogs.StartPrint.DoYouWantToStartFilename', { filename: file.filename }) }}
+                <p class="body-2">
+                    {{ question }}
+                </p>
             </v-card-text>
-            <template v-if="moonrakerComponents.includes('timelapse')">
-                <v-divider class="mt-3 mb-2"></v-divider>
-                <v-card-text class="pb-0">
+            <start-print-dialog-spoolman v-if="moonrakerComponents.includes('spoolman')" :file="file" />
+            <template v-if="moonrakerComponents.includes('timelapse') || true">
```

Comment:
constant true makes timelapse always visable

Reply:
ohh. forgot to delete this from testing...